AGENTS.md

```
@@ -3,23 +3,38 @@
 ## Build/Lint/Test Commands
 
 ### Core Commands
-- `npm run test` - Run Jest tests + validate all 70+ commands
-- `npm run test:jest` - Run Jest tests only
-- `npm run test:validate` - Run command validation only
-- `npm run validate` - Validate all 70+ commands (expects exact count)
-- `npm run lint` - Run markdownlint on documentation
-- `npm run lint:fix` - Auto-fix markdown formatting issues
-- `npm run ci` - Full pipeline: validate + lint + link-check
+- `bun run test` - Run Jest tests + validate all 70+ commands
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)